plugin.video.repubblicaitaliana/addon.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.repubblicaitaliana" name="RepubblicaItaliana" version="1.0.0" provider-name="Testato">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="inputstream.adaptive" version="2.6.7"/>
+    <import addon="plugin.video.youtube" version="6.8.10+matrix.1"/>
```

Comment:
You might want to rely on tubed for matrix in the future, it's the new version of YouTube for kodi

Reply:
How i can swap from youtube to tubed ?

Now i call the video in this way:
link = 'plugin://plugin.video.youtube/play/?video_id=_pjPv7dS-_w'

I tried:
link = 'plugin://plugin.video.tubed/play/?video_id=_pjPv7dS-_w'

But it do not work.
Sorry if I ask here but I asked on Kodi forum without answer

Thanks

Reply:
See here:
https://github.com/XBMC-Addons/plugin.video.nasa/commit/169032edcdf71ffda247ad2a605824df7018d25b